modules/components/src/Arranger/Aggregations.js

```diff
@@ -18,6 +18,12 @@ const Aggregations = ({
   api,
   Wrapper = BaseWrapper,
   containerRef,
+  componentProps = {
+    getTermAggProps: () => ({}),
+    getRangeAggProps: () => ({}),
+    getBooleanAggProps: () => ({}),
+    getDatesAggProps: () => ({}),
+  },
```

Comment:
just made these functions so in case we need to switch props logically based on different fields or something, then we can pass it in from within `aggComponentsMap`